games-simulation/flightgear/flightgear-2018.2.2.recipe

```diff
@@ -0,0 +1,84 @@
+SUMMARY="The FlightGear Flight Simulator"
+DESCRIPTION="The goal of the FlightGear project is to create a sophisticated \
+and open flight simulator framework for use in research or academic environments, \
+pilot training, as an industry engineering tool, for DIY-ers to pursue their favorite \
+interesting flight simulation idea, and last but certainly not least as a fun, \
+realistic, and challenging desktop flight simulator. We are developing a sophisticated, \
+open simulation framework that can be expanded and improved upon by anyone interested \
+in contributing."
+HOMEPAGE="https://www.flightgear.org/"
+COPYRIGHT="2002-2018 Curtis Olsen and others"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/flightgear/flighgear-$portVersion.tar.bz2"
+CHECKSUM_SHA256="924bcaec4bead47d01638f1033c0ca95c4a3fff65bd43d9c5b171d354b4b28f6"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+ADDITIONAL_FILES="flightgear.rdef.in"
```

Comment:
```diff
 CHECKSUM_SHA256="924bcaec4bead47d01638f1033c0ca95c4a3fff65bd43d9c5b171d354b4b28f6"
+ADDITIONAL_FILES="flightgear.rdef.in"
 
 ARCHITECTURES="!x86_gcc2 x86 x86_64"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
 SECONDARY_ARCHITECTURES="x86"
+fi
-ADDITIONAL_FILES="flightgear.rdef.in"
 
PROVIDES="
 	flightgear$secondaryArchSuffix
-	cmd:fgcom$secondaryArchSuffix
+	cmd:fgcom
-	cmd:fgelev$secondaryArchSuffix
+	cmd:fgelev
```
(The `$secondaryArchSuffix` should be removed from all the other `cmd:` line in `PROVIDES`.)